ext/standard/php_password.h

```diff
@@ -29,10 +29,22 @@ PHP_MSHUTDOWN_FUNCTION(password);
  * When updating these values, synchronize ext/sodium/sodium_pwhash.c values.
  * Note that libargon expresses memlimit in KB, while libsoidum uses bytes.
  */
+#if defined(PHP_SODIUM_PWHASH_MEMLIMIT)
+#define PHP_PASSWORD_ARGON2_MEMORY_COST PHP_SODIUM_PWHASH_MEMLIMIT
```

Comment:
Why these changes are needed? If it's for consistency with php_libsodium.h, then you can get rid of these added lines. `#if defined(PHP_SODIUM_PWHASH_MEMLIMIT)` should never be `true` at that point, because this line defines it.

Reply:
Yes, it was for consistency (based on the comments above). I wasn't sure whether the header file of `php_libsodium` could be included before that of `password` so I added these lines to be on the safe side. I'll remove them shortly.